src/main/java/in/org/projecteka/hiu/consent/PatientConsentRepository.java

```diff
@@ -1,36 +1,33 @@
 package in.org.projecteka.hiu.consent;
 
-import in.org.projecteka.hiu.consent.model.PatientConsentRequest;
-import io.vertx.core.json.JsonObject;
 import io.vertx.pgclient.PgPool;
 import io.vertx.sqlclient.Tuple;
 import lombok.AllArgsConstructor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import reactor.core.publisher.Mono;
 
+import java.time.LocalDateTime;
 import java.util.UUID;
 
 import static in.org.projecteka.hiu.ClientError.dbOperationFailure;
-import static in.org.projecteka.hiu.common.Serializer.from;
 
 @AllArgsConstructor
 public class PatientConsentRepository {
 
     private static final Logger logger = LoggerFactory.getLogger(PatientConsentRepository.class);
 
     private static final String INSERT_PATIENT_CONSENT_REQUEST = "INSERT INTO " +
-            "patient_consent_request (patient_consent_request, data_request_id, hip_id) VALUES ($1, $2, $3)";
+            "patient_consent_request (data_request_id, hip_id) VALUES ($1, $2)";
```

Comment:
are you not supposed to also save the patient_id?

Reply:
that is not part of this PR no. that will added along with the other changes